example/spec/scaling/scalingSpec.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+const find = require('lodash.find');
+
+const { autoscaling, ecs } = require('@cumulus/common/aws');
+const {
+  buildAndStartWorkflow,
+  waitForCompletedExecution,
+  getClusterArn
+} = require('@cumulus/integration-tests');
+const { sleep } = require('@cumulus/common/util');
+const { loadConfig, loadCloudformationTemplate } = require('../helpers/testUtils');
+
+const workflowName = 'HelloWorldOnDemandWorkflow';
+const config = loadConfig();
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000000;
+let clusterArn;
+let autoScalingGroupName;
+
+async function getClusterStats({
+  statTypes = ['runningEC2TasksCount', 'pendingEC2TasksCount']
+}) {
+  const stats = (await ecs().describeClusters({
+    clusters: [clusterArn],
+    include: ['STATISTICS']
+  }).promise()).clusters[0].statistics;
+  const returnedStats = {};
+  statTypes.forEach((statType) => {
+    returnedStats[statType] = parseInt(find(stats, ['name', statType]).value, 10);
+  });
+  return returnedStats;
+}
+
+async function getAutoScalingGroupName(stackName) {
+  const autoScalingGroups = (await autoscaling().describeAutoScalingGroups({}).promise()).AutoScalingGroups;
+  const asg = find(autoScalingGroups, (group) => group.AutoScalingGroupName.match(new RegExp(stackName, 'g')));
+  return asg.AutoScalingGroupName;
+}
+
+const waitPeriod = 30000;
+async function getNewScalingActivity() {
+  const params = {
+    AutoScalingGroupName: autoScalingGroupName,
+    MaxRecords: 1
+  };
+  let activities = await autoscaling().describeScalingActivities(params).promise();
+  const startingActivity = activities.Activities[0];
+  let mostRecentActivity = Object.assign({}, startingActivity);
+  /* eslint-disable no-await-in-loop */
+  while (startingActivity.ActivityId === mostRecentActivity.ActivityId) {
+    activities = await autoscaling().describeScalingActivities(params).promise();
+    mostRecentActivity = activities.Activities[0];
+    console.log(`No new activity found. Sleeping for ${waitPeriod / 1000} seconds.`);
+    await sleep(waitPeriod);
+  }
+  /* eslint-enable no-await-in-loop */
+
+  return mostRecentActivity;
+}
+
+const numExecutions = 3;
+let cloudformationResources;
+let numActivityTasks;
+let minInstancesCount;
+
+describe('cloudformation template for scaling policies', () => {
```

Comment:
I'm finding myself questioning the utility of these tests.  Thoughts?

Reply:
I think they are a useful sanity check of the config.yml + cloudformation template yml logic